### PR TITLE
Game reset fix

### PR DESCRIPTION
**Pull request by KmBarnett**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -15,10 +15,11 @@
   var rangeView = document.querySelectorAll('.range-view-js');
   var updateButton = document.querySelector('#update-button');
   var inputForRange = document.querySelector('#set-range-box');
-  var randomNumber = createNumber();
   var winningSide = document.querySelector('#winning-side');
+  var gameResetButton = document.querySelector('#reset-button');
   var numberOfGuesses = 0;
-  var deleteWinnerCard
+  var deleteWinnerCard;
```
Comment (Karlfunhouse) [inline main.js:21]: why did you rearrange the variables like this?

Comment (KmBarnett) [reply main.js:21]: Just to have the querySelector vars together.

File: main.js [hunk 2]
```diff
@@ -30,6 +31,19 @@
     clearButton.setAttribute('disabled', true);
   }
 
+  function resetGame() {
```
Comment (Karlfunhouse) [inline main.js:34]: this is iteration 1.6 that resets the guess form when the winner is declared and resetting the min range to 0 and max range to 100 and then clears the fields for the next game

File: main.js [hunk 3]
```diff
@@ -43,9 +57,11 @@
   function setRange() {
     rangeView[0].innerText = minRange.value;
     rangeView[1].innerText = maxRange.value;
-    var min = parseInt(minRange.value)
-    var max = parseInt(maxRange.value)
+    var min = parseInt(minRange.value || 1);
```
Comment (Karlfunhouse) [inline main.js]: I don't see why we would need these lines of code.   Isn't the min value already default of 1 and max value of 100?

File: main.js [hunk 4]
```diff
@@ -104,22 +121,22 @@
 
   function checkGuess1() {
     if (parseInt(challengerGuesses[0].value) < randomNumber) {
-      guessHint[0].innerText = 'Too Low!'
+      guessHint[0].innerText = 'Too Low!';
```
Comment (Karlfunhouse) [inline main.js:124]: semi-colon magic!!!! Good jorb.

File: styles.css [hunk 5]
```diff
@@ -228,7 +228,7 @@ h4 {
   padding: 5px 0;
   align-content: center;
   border-color: #aeaeae;
-  display: relative;
+  text-transform: uppercase;
```
Comment (Karlfunhouse) [inline styles.css:231]: Winner with a Capital WIN

File: styles.css [hunk 6]
```diff
@@ -309,11 +305,11 @@ h4 {
 .close-button {
   background-image: url('x-mark-3-512.png');
   background-size: 100%;
+  background-repeat: no-repeat;
```
Comment (Karlfunhouse) [inline styles.css:308]: Is this for when the x button would repeat itself when the screen would get too big?

**Review by Karlfunhouse — COMMENTED** [review]
Everything looks pretty good, just take a look at my comments and see if they make sense for resetting the min/max values using || or if we can cut that out.

File: main.js (hunk 3 above)
Comment (KmBarnett) [reply main.js]: No I hard coded them instead

File: styles.css (hunk 6 above)
Comment (KmBarnett) [reply styles.css:308]: so when the screen shrinks to much it doesn't repeat the  background image we used

File: main.js (hunk 1 above)
Comment (Karlfunhouse) [reply main.js:21]: Okay cool.

File: styles.css (hunk 6 above)
Comment (Karlfunhouse) [reply styles.css:308]: ✨

**Review by Karlfunhouse — APPROVED** [review]

**Pull request merged**